dappnode_package.json

```diff
@@ -1,33 +1,31 @@
 {
-  "name": "storj.public.dappnode.eth",
-  "version": "0.0.8",
-  "upstreamVersion": "v1.6.4",
-  "shortDescription": "Earn tokens by sharing your disk space on the Storj network",
-  "description": "Storj is the supply-side node for offering hard drive capacity and bandwidth to their suite of decentralized applications. Storj pays you for your unused disk space and connection in **STORJ tokens!**",
-  "type": "service",
-  "author": "edu",
-  "architectures": ["linux/amd64", "linux/arm64"],
-  "backup": [
-    {
-      "name": "identity",
-      "path": "/app/identity"
-    }
-  ],
-  "globalEnvs": {
-    "all": true
-  },
-  "categories": [
-    "Economic incentive"
-  ],
-  "links": {
-    "Dashboard": "http://storj.public.dappnode/"
-  },
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/dappnode/DAppNodePackage-storj.git"
-  },
-  "bugs": {
-    "url": "https://github.com/dappnode/DAppNodePackage-storj/issues"
-  },
-  "license": "GPL-3.0"
+	"name": "storj.public.dappnode.eth",
+	"version": "0.0.8",
+	"upstreamVersion": "v1.6.4",
+	"shortDescription": "Earn tokens by sharing your disk space on the Storj network",
+	"description": "Storj is the supply-side node for offering hard drive capacity and bandwidth to their suite of decentralized applications. Storj pays you for your unused disk space and connection in **STORJ tokens!**",
+	"type": "service",
+	"author": "edu",
+	"architectures": ["linux/amd64", "linux/arm64"],
+	"backup": [
+		{
+			"name": "identity",
+			"path": "/root/.local/share/storj/identity/storagenode"
+		}
+	],
+	"globalEnvs": {
+		"all": true
+	},
+	"categories": ["Economic incentive"],
+	"links": {
+		"ui": "http://storj.public.dappnode/"
+	},
+	"repository": {
+		"type": "git",
+		"url": "https://github.com/dappnode/DAppNodePackage-storj.git"
+	},
+	"bugs": {
+		"url": "https://github.com/dappnode/DAppNodePackage-storj/issues"
+	},
+	"license": "GPL-3.0"
```

Comment:
You have format the manifest to 2 spaces per tab, since that's what the SDK will format it to eventually